spring/spring-boot-session-replication-spring-session-hazelcast/src/main/java/com/hazelcast/guide/controller/SessionController.java

```diff
@@ -92,18 +92,28 @@ public String getSessionInfo(HttpServletRequest request) {
     }
 
     private String toHtmlTable(Map<String, Object> attributes) {
-        StringBuilder html = new StringBuilder("<html>");
-        html.append("<table border=\"1\" cellpadding=\"5\" cellspacing=\"5\">");
-        attributes.forEach((k, v) -> addHtmlTableRow(html, k, v));
-        html.append("</table></html>");
-        return html.toString();
+        String html = """
+                <html>
+                    <body>
+                        <table style="border-spacing: 5px; border: 1px solid black;">
+                        %s
+                        </table>
+                    </body>
+                </html>
+                """;
+        String rows = attributes.entrySet().stream()
+                                .map(e -> addHtmlTableRow(e.getKey(), e.getValue()))
+                                .collect(Collectors.joining("\n"));
```

Comment:
```suggestion
                                .collect(Collectors.joining("%n"));
```

But `System.lineSeparator()` would be clearer IMO.